### PR TITLE
List sort on the basis of tuple

### LINKED ISSUE #5711: Ascending order of list in python
## 🚀 Feature

(A clear and concise description of what the feature is.)
I want to implement a program in python which will arrange a list in ascending order on the basis of the last element in each tuple from a given list.

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?
Yes

## Pitch

(Please explain why this feature should be implemented and how it would be used.)
It would be help in arranging the list when tuple is given.

### PR DESCRIPTION
**Pull request by Khushi-agarwal**
Fixes #5711 
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)
Yes

### Description

It is a python program which is used to sort list having tuples in ascending order on the basis of last element of tuple.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)
The PR solves the issue #5711

### CODE REVIEW

**Comment by Khushi-agarwal** [comment]
@bhav09 @ankitaggarwal23 kindly review it

File: Python/sort/ListTuplesort.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+#Here li will store the list of reversed tuples
+li=lys(ls) 
+#It will sort the list in ascending order
+li.sort() 
+#Again we call the function lys so that after sorting the elements get arranged back
+print(lys(li))
+
+#This is a function which is used to reverse the elements of tuples 
+#It returns a list of reversed tuples
+def lys(ls):
```
Comment (bhav09) [inline Python/sort/ListTuplesort.py:9]: Function definition should always be above driver code.

File: Python/sort/ListTuplesort.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+'''
+It is a python program to get a list having tuples contained,
+to be sorted in ascending order on the basis of the last element 
+in each tuple of a list
+'''
+
+#Driver's code
+#It is used to take input a list from user which will contain tuples
+#It makes use of map function as it simplifies the process
+ls=list(tuple(map(int,input().split()))for r in range(int(input('enter number of elements'))))
```
Comment (bhav09) [inline Python/sort/ListTuplesort.py]: Can you use a main function in the driver code

File: Python/sort/ListTuplesort.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+print(lys(li))
+
+#This is a function which is used to reverse the elements of tuples 
+#It returns a list of reversed tuples
+def lys(ls):
+    return [tuple(reversed(tup)) for tup in ls]
+
+
+'''
+Input/Output-
```
Comment (bhav09) [inline Python/sort/ListTuplesort.py:29]: Need to add complexity above Sample IO

File: Python/sort/ListTuplesort.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+Enter number of elements 5
+5 6 
+7 8
+6 7
+1 2
+3 4 
+[(1, 2), (3, 4), (5, 6), (6, 7), (7, 8)]
+'''
```
Comment (bhav09) [inline Python/sort/ListTuplesort.py]: Add a new line at EOF

**Review by bhav09 — CHANGES_REQUESTED** [review]
Reviewed.

**Comment by Khushi-agarwal** [comment]
@bhav09 I have made the required changes.Kindly review.

**Pull request merged**